packages/react/src/components/TooltipDefinition/TooltipDefinition.js

```diff
@@ -77,6 +77,7 @@ const TooltipDefinition = ({
         type="button"
         className={tooltipTriggerClasses}
         aria-describedby={tooltipId}
+        aria-label={tooltipText}
```

Comment:
Would this end up causes the same text (`tooltipText`) to be announced twice from both `aria-label` and `aria-describedby`

Reply:
@joshblack good point! `aria-describedby` announces the `tooltipId` which I think should be different from the text

Reply:
@andreancardona I believe the way `aria-describedby` works is that it uses the `id` to find the DOM node from which it computes the description. In this case, using `aria-label` with the `tooltipText` would give the node the accessible name of `tooltipText`. It would also give the node the description of `tooltipText` from `aria-describedby` as it points to a node that contains that as its accessible label.

In the preview, we can see this in the accessibility tab: 

![image](https://user-images.githubusercontent.com/3901764/108538772-c594d300-72a4-11eb-9565-085b24e5c05e.png)

Another issue that this can cause is that now the `button` no longer uses the underlying text as the label, so the user would only hear the tooltip text but would not know what that text it is defining.